src/components/Display.js

```diff
@@ -1,10 +1,14 @@
+/* eslint-disable react/prop-types */
+/* eslint-disable react/destructuring-assignment */
+/* eslint-disable react/jsx-filename-extension */
+/* eslint-disable react/prefer-stateless-function */
 import React from 'react';
 
 
-class Display extends React.Component{
-  render(){
-    return(
-      <div class="calculator__display">
+class Display extends React.Component {
+  render() {
+    return (
+      <div className="calculator__display">
         <p>{this.props.childDisplay}</p>
       </div>
     );
```

Comment:
"We can use a function component to render it since it is stateless." => I agree but isn't it too early in the review to do that? I can't see where we were told that it should be stateless though. I think all these are based on personal style. 
"Let destructure the childDisplay props in the parameter." => I have another approach to this.I think if I used props I will not be penalised just like I did for another component in the code? 

"Using PropType let's give the childDisplay prop a string type and a default value of 0. Learn how to use prop-types here" => is it mandatory I use PropTypes? I know they exist but don't I have a choice on what I want to use and how I intend to achieve the task? just a question which needs a little bit of clarity. 

"Also, let use result instead of childDisplay, as this is more descriptive than childDisplay." => I saw that in the instruction and my thoughts was that I'd come to them later, so can I come to this later?

Reply:
> I agree but isn't it too early in the review to do that? I can't see where we were told that it should be stateless though. I think all these are based on personal style.

It's not too early. This is the milestone dedicated to this.
I am addressing the linter you disabled hence not a personal style 

>  is it mandatory I use PropTypes? I know they exist but don't I have a choice on what I want to use and how I intend to achieve the task? just a question which needs a little bit of clarity.

It is required to give it a type of `String` and a default value of `0` as in the requirement, please note that this has not been achieved.

> I saw that in the instruction and my thoughts was that I'd come to them later, so can I come to this later?

Nope, let's do that now.

Reply:
Bros, I think you need to suggest to Emily/Microverse (I will do that as well) that the requirements for this project should be crafted in such a way that it is more intuitive to students. Why do I say this? I have done React before and I must say the code syntax may not be professional but I understand the basics of React and how it works. Then I did not have to do Type checking and it's a big Gotcha in this milestone. So in order to reduce all these Gotchas there is need for microverse to recraft the requirements for this project such that students know intuitively what to do. For example in this milestone student should be told explicitly that props passed to components need to be typed checked. If this is done it will reduce your workload when reviewing codes. Writing all this review about the `Display` component and other components could be summarised as you this "You forgot to type check your props" and then you can attach a link for the student to know what type checking is eventually.

Reply:
I did not see the requirements where they asked us to use "propTypes". They only mentioned props. It would have been better if they said "propTypes" explicitly and that is the basis of my submission not telling me this: "use props with a specific type". If I may ask is this intuitive? No I'd say. Not all students will know prop type immediately they are told. Did you know about prop type immediately you did the curriculum?